package.json

```diff
@@ -22,7 +22,7 @@
     "grunt-contrib-uglify": "^5.2.2",
     "grunt-sass": "^3.1.0",
     "grunt-svg-sprite": "^2.0.2",
-    "jquery": "1.12.4",
+    "jquery": "^3.7.1",
```

Comment:
Are we missing a commit for the lockfile? NPM library versions are determined from the lockfile and the PR doesn't have a lockfile update.